nose_parameterized/parameterized.py

```diff
@@ -211,11 +211,53 @@ def expand(cls, input, testcase_func_name=None):
             >>>
             """
 
-        def parameterized_expand_wrapper(f):
+        def doc_expansion(f, args, kwargs, is_method):
+            offset = 1 if is_method else 0
+            vns = inspect.getargspec(f)[0][offset:]
+            descs = ["{0} = {1}".format(n, repr(v)) for n, v in zip(vns, args)]
```

Comment:
Noted.

I'll add some mixed-unicode testcases too. Unicode is just annoyingly painful to get right a lot of the time :(